include/coap2/encode.h

```diff
@@ -42,7 +42,13 @@ extern int coap_flsll(long long i);
 #endif
 
 /* ls and s must be integer variables */
-#define COAP_PSEUDOFP_ENCODE_8_4_DOWN(v,ls) (v < HIBIT ? v : (ls = coap_fls(v) - Nn, (v >> ls) & MMASK) + ls)
```

Comment:
Hm, nowadays, this macro seems to be used in tiny.c only, and `COAP_PSEUDOFP_ENCODE_8_4_UP` is not used anymore at all. How about moving the (now inline) function to tiny.c and get rid of the dead code in the core library?

Reply:
Moved all the COAP_PSEUDOFP_stuff from encode.h to tiny.c.  Updated code pushed.